ansible_collections/arista/avd/docs/how-to/tower-integration.md

```diff
@@ -208,12 +208,11 @@ collections:
     version: 2.1.0
 ```
 
-
 ### What to change to work with AVD and AWX
 
 Ansible has a default variable that point to inventory file used in playbook and named `{{ inventory_file }}`. Since AWX/Tower is using a database, this variable is not available anymore and [inventory file does not exist in such environment](https://github.com/ansible/awx/issues/5926).
 
-AVD use this variable to read inventory and to build container topology on Cloudvision. So to mitigate this behavior, a small warkaround is to add a task that download your inventory from your git repository and define `{{ inventory_file }}`:
+AVD use this variable to read inventory and to build container topology on Cloudvision. So to mitigate this behavior, a small workaround is to add a task that download your inventory from your git repository and define `{{ inventory_file }}`:
```

Comment:
Change to: AVD uses this variable to read the inventory and to build a container topology on Cloudvision. So to mitigate this behavior, a small workaround is to add a task that downloads your inventory from your git repository and define `{{ inventory_file }}`: